test/python/providers/test_backend_v2.py

```diff
@@ -20,25 +20,35 @@
 from ddt import ddt, data
 
 from qiskit.circuit import QuantumCircuit, ClassicalRegister, QuantumRegister
-from qiskit.compiler import transpile
-from qiskit.providers.fake_provider import FakeMumbaiFractionalCX
-from qiskit.providers.fake_provider.fake_backend_v2 import (
-    FakeBackendV2,
-    FakeBackend5QV2,
-    FakeBackendSimple,
-    FakeBackendV2LegacyQubitProps,
+from qiskit.circuit.library.standard_gates import (
+    CXGate,
+    ECRGate,
 )
-from qiskit.providers.fake_provider.backends import FakeBogotaV2
+from qiskit.compiler import transpile
+from qiskit.transpiler import InstructionProperties
+from qiskit.providers.fake_provider import FakeMumbaiFractionalCX, GenericBackendV2
+from qiskit.providers.basic_provider import BasicSimulator
 from qiskit.quantum_info import Operator
 from qiskit.pulse import channels
 from test import QiskitTestCase  # pylint: disable=wrong-import-order
 
+from ..legacy_cmaps import BOGOTA_CMAP, TENERIFE_CMAP
```

Comment:
Minor, but I think in most of our test files, we use absolute imports for other things in the test package (e.g. `from test.python.`) which has the benefit of letting us move files around in subdirectories without requiring a code change.

Reply:
I tried to avoid the absolute import here because pylint complains about the import order when importing from `test` (see the pylint disable in line 33), and I didn't want to keep disabling it, but I'm fine changing it too.

Reply:
Ah alright, that's unfortunate, but fair enough 😬. I'm fine with leaving it as is.